wiki/README.md

```diff
@@ -20,3 +20,4 @@ In order to run the `po4a-update-templates.sh` and `po4a-create-all-targets.sh`
 
 - If you want something displayed on a new line immediately below one that does not end with a period, the second line must start with a space; e.g. if you want a line break after } or ). For code blocks introduced with ``` or ~~~, these characters must be followed by a line break (enter) and a space at the start of the next line. The closing formatting characters should be on their own line too.
 - Any corrections made to languages other than EN **must** be done to their .po files. Directly editing .md files won't break anything but any changes done that way will be lost the next time `po4a-create-all-targets.sh` is run.
+- Headings introduced with # must have double spacing before the text body.
```

Comment:
Is this something we can check for in linting? I'm sure I'd not be able to remember to do this,

Reply:
I must admit the wording is a bit messy and could do with a review. These started out as "notes to self" and might not be fully accurate. There were I think 4 examples where po4a messed up the formatting and it was always related to how it interprets where a paragraph ends. If you want I could provide the specific examples to make it clearer.

Reply:
Just to add - I cloned your adm-man branch and processed all the wiki/en files with po4a to see if any of the new stuff came out wrong. It's all fine, with no weird formatting, so I don't think it's going to be too much of an issue.